addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18404,7 +18404,31 @@ msgctxt "#36050"
 msgid "On start"
 msgstr ""
 
-#empty strings from id 36051 to 36098
+#empty strings from id 36051 to 36094
+
+#. Label of setting "System / Display / Use 10-bit surfaces"
+#: system/settings/settings.xml
+msgctxt "#36095"
+msgid "Use 10-bit surfaces"
```

Comment:
As this is only for SDR and probably most uses doesn't know what a surface is or used for, perhaps change to something like:

Use for 10 bit for SDR

Reply:
Maybe:

Use 10-bit video surfaces for SDR